spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/ArmeriaSecurityAutoConfiguration.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.spring;
+
+import static com.linecorp.armeria.internal.spring.ArmeriaConfigurationUtil.configurePorts;
+
+import java.net.InetSocketAddress;
+import java.util.Optional;
+
+import javax.annotation.Nullable;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.ConfigurableEnvironment;
+import org.springframework.core.env.PropertySource;
+import org.springframework.util.SocketUtils;
+
+import com.google.common.base.Strings;
+import com.google.common.collect.ImmutableList;
+
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.SessionProtocol;
+import com.linecorp.armeria.server.DecoratingServiceBindingBuilder;
+import com.linecorp.armeria.server.ServerBuilder;
+import com.linecorp.armeria.spring.ArmeriaSettings.Port;
+
+/**
+ * Spring Boot {@link Configuration} that provides Armeria integration.
+ */
+@Configuration
+@EnableConfigurationProperties(ArmeriaSettings.class)
+public class ArmeriaSecurityAutoConfiguration {
+
+    private static final Port DEFAULT_SECURITY_PORT = new Port().setPort(8001)
```

Comment:
Question: Where is the `8001` from? Spring uses it?

Reply:
I think we shouldn't use the default port.
If a use forgets to disable the security, an attacker might use the default port to get the information of the server. What do you think?

Reply:
I understood [here](https://github.com/line/armeria/pull/2502#discussion_r388317746) to mean adding a default port.

Reply:
If so, should this feature be enabled only by adding a spring-actuator?

Reply:
I removed it. :)